grpc/execution/server_test.go

```diff
@@ -170,17 +170,171 @@ func TestExecutionServiceServerV2_ExecuteBlock(t *testing.T) {
 		},
 	}
 
-	fork := ethservice.BlockChain().Config().AstriaForks.GetForkAtHeight(1)
-	var bridgeConfig params.AstriaBridgeAddressConfig
-	for _, cfg := range fork.BridgeAddresses {
-		bridgeConfig = *cfg
-		break
+	for _, tt := range tests {
```

Comment:
github is being weird about this diff. I moved the fork logic in otherwise the deposits were never executed because we generate new bridge addresses with each test run, the bridge addresses don't match. We weren't validating that all the txs executed before, I made sure that is happening.

Added a new test flow where runs with low gas to ensure the deposits execute even when other txs cannot.